### ISSUE TITLE
Oppdatere archimate-dokumentasjon

### ISSUE THREAD

**Issue by xivind** [hdir/strava-club#23]
Må komme frem at databehandler også leser results.json, ikke bare produserer den

**Comment by thomiz** [comment]
Oppdaterte dokumentasjonen slik at det blir riktige filnavn og les/skriv fra results.json

**Comment by xivind** [comment]
Ser bra ut, men den club_members.csv-filen leser vi ikke nå, så kanskje fjerne relasjonen? For øvrig kan kanskje navnet på de tre forretningstjenestene være Hente data, Prosessere data og den siste kan være som den er Presentere? For det er jo det som er det interessante fra forretningsståstedet?


![Image](https://github.com/hdir/strava-club/assets/8987107/ac0e1d1a-6a1a-4015-b4eb-763c859ec58d)

**Comment by thomiz** [comment]
Dessuten mangler det oppdatering av teksten for hver komponent, kanskje du kan se på teksten @xivind ?

**Comment by xivind** [comment]
Yes, den teksten skal jeg se på @thomiz . Kommer tilbake til deg..

**Comment by thomiz** [comment]
Jeg har også fikset buggen i archifiguren @xivind

**Comment by xivind** [comment]
Tusen takk @thomiz  Jeg lar denne stå åpen fortsatt, siden det er noe utestående på meg her. Jeg må oppdatere teksten for hver komponent.

**Comment by xivind** [comment]
Readme.md og komponentbeskrivelsene er oppdatert. Tar du en titt @thomiz ? PS For konsistens, kanskje boksen som viser til scraper burde hete "Skraper" og ikke filnavnet på skraper, slik at det er likt med slik vi navngir databehandler og presentatør? Eventuelt bør de to sistnevnte også refereres med filnavn. Bestem du @thomiz hva du synes er best. Jeg synes modulnavn er bedre enn filnavn. Ellers anser jeg denne som klar fra min kant.

![Image](https://github.com/hdir/strava-club/assets/8987107/80cd932f-df87-4fc9-abf4-ee92ddcd4f47)

**Comment by thomiz** [comment]
Vurderes senere
